### ISSUE TITLE
"Winget is not installed"

### ISSUE THREAD

**Issue by erik-the-blue** [ChrisTitusTech/winutil#1680]
**Describe the bug**
I think this may be a bug? It only started happening after I removed all onedrive folders and renamed my user folder in Windows 11. Problem persisted after redownloading winutil again. Every time I click "Get Installed" it says "winget is not installed", but it is installed. I even went to the config tab and clicked "reinstall winget". I fully let it reinstall, but get the same response when clicking "Get Installed" on the install tab.

Included screenshot of the 'error'. I'm not sure it's actually a bug or a result of what I changed, but I have no idea why it persists, or how to remedy it. First time making a report here, apologies if anything is wrong.

**Screenshots**
![Screenshot 2024-03-05 111419](https://github.com/ChrisTitusTech/winutil/assets/162367024/eeca2e6d-a4c5-4fe5-8c6c-3a3faf79dac5)

**Comment by Marterich** [comment]
Could you please run the command "winget --version" in the Win11 Terminal or Powershell and comment the output of the command so we can see which winget version (if any) is installed on your system :)

**Comment by erik-the-blue** [comment]
> Could you please run the command "winget --version" in the Win11 Terminal or Powershell and comment the output of the command so we can see which winget version (if any) is installed on your system :)

So I ran a few winget commands even before I made this post, and unfortunately I guess my problem must be a much larger issue. I ran it again for the screenshot. I also downloaded the winget installer from the Microsoft Github, and took a screenshot of it saying I have it installed already. I'm beginning to believe this might just be a corrupted user profile? I guess it's not a winutil issue.

![Screenshot 2024-03-06 100158](https://github.com/ChrisTitusTech/winutil/assets/162367024/af1aa59a-67be-499a-9188-6691cf5d2ccd)
![Screenshot 2024-03-06 100401](https://github.com/ChrisTitusTech/winutil/assets/162367024/43ac9015-a55e-451b-9eef-f9aa16cdf98e)

**Comment by KALIXHOSTING** [comment]
Yes having the same issue I installed via the microsoft store then check in powershell winget --version and the tool started to install perfectly but something is definitely up with the tool on how its grabbing winget and think choco install winget is not working problem either

**Comment by erik-the-blue** [comment]
To update anyone viewing this in the future, I'm pretty sure I fixed this issue, and I suppose it wasn't a winutil issue after all. 
**All I did was reset and restart windows indexing**. It took waiting for a full index to recomplete, but now the winget commands are working, as is the "get installed" function in winutil. Closing the issue.

![Screenshot 2024-03-08 140900](https://github.com/ChrisTitusTech/winutil/assets/162367024/8065aa10-da94-4693-a36e-f073860d867c)
![Screenshot 2024-03-08 141233](https://github.com/ChrisTitusTech/winutil/assets/162367024/822e9a7e-b1d4-4a46-a860-537d1c998804)
